staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -130,6 +132,9 @@ type GenericAPIServer struct {
 	swaggerConfig *swagger.Config
 	openAPIConfig *openapicommon.Config
 
+	// Enables updating OpenAPI spec using update method.
+	OpenAPIService *openapi.OpenAPIService
```

Comment:
Do we use a normal API server code path for serving Custom Resources? I heard something about those resources can have dynamic OpenAPI spec, right?